src/editor/inspect.lua

```diff
@@ -37,17 +37,26 @@ if ide.config.staticanalyzer.luacheck then
   end
 
   warnings_from_string = function(src, file)
-    local data = luacheck.check_strings({src}, config.options or {
+    local api_globals = build_env()
+
+    if config.options then
+      -- add user config globals to api table
+      for k, v in pairs(config.options.globals) do
```

Comment:
Is `config.options.globals` guaranteed to exist? I'd add a ` or {}` just to be sure.

Reply:
Very good point, thanks - changing!